js/src/common/components/SelectDropdown.js

```diff
@@ -21,7 +39,7 @@ export default class SelectDropdown extends Dropdown {
   }
 
   getButtonContent(children) {
-    const activeChild = children.filter((child) => child.attrs.active)[0];
+    const activeChild = children.filter(isActive)[0];
```

Comment:
We can now use `Array.prototype.find()` because we're screwing IE. 🎉

Reply:
Lol! https://github.com/flarum/core/pull/2255/commits/ab9d72d860787f9b89fbab80d5e90ba05566c459